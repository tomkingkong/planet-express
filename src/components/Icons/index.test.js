import React from 'react';
import { shallow } from 'enzyme';

import { PlanetIcon } from '.';

describe('PlanetIcon component', () => {
  it('should match planetIcon snapshot', () => {
    const wrapper = shallow(<PlanetIcon planetMap={'images/alternate/exoplanet_habitable_1.png'} />);
    expect(wrapper).toMatchSnapshot();
  });
});