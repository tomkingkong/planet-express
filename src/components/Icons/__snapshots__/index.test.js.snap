// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlanetIcon component should match planetIcon snapshot 1`] = `
<div
  className="planet__wrapper"
>
  <div
    className="planet"
    style={
      Object {
        "backgroundImage": "url(images/alternate/exoplanet_habitable_1.png)",
      }
    }
  />
</div>
`;
